Tasks/Common/nuget-task-common/NuGetToolRunner2.ts

```diff
@@ -41,31 +45,59 @@ function prepareNuGetExeEnvironment(
             }
         }
 
+        // New credential provider
+        if (e.toUpperCase() === "NUGET_PLUGIN_PATHS") {
+            pluginCredentialProviderPath = input[e];
+            continue;
+        }
+
+        // Old credential provider
         if (e.toUpperCase() === "NUGET_CREDENTIALPROVIDERS_PATH") {
             originalCredProviderPath = input[e];
+            continue;
+        }
 
-            // will re-set this variable below
+        if (e.toUpperCase() === "VSS_NUGET_URI_PREFIXES") {
+            prefix = input[e];
```

Comment:
I don't think I should add this (I added it originally for testing devfabric because I had issues with the prefixes)

Reply:
Yes, the prefixes should remain the same.